keps/2724-topology-aware-scheduling/README.md

```diff
@@ -621,6 +622,20 @@ for [Beta](#beta). The initial approach for the design is left in the
 [Support for ReplicatedJobs in JobSet](#support-for-replicatedjobs-in-jobset)
 section.
 
+### Implicit defaulting of TAS annotations
+
+Requiring to set the TAS annotations (see [User facing API](#user-facing-api))
+on every PodSet creates a friction for adoption of TAS, and a point of failure.
+
+In order to reduce friction we assume that TAS is used for scheduling workloads
+targetting CQs with only TAS Resource Flavors (RFs with the `spec.topologyName`
+field specified).
+
+A PodSet scheduled with TAS without the explicit annotation is handled as
+if it had the `podset-preferred-topology` annotation pointing to the lowest
+topology level defined in the Topology referenced by the selected TAS flavor.
```

Comment:
Add some explanation what will TAS algorithm do if preferred topology is set to the lowest level.